e2e/autoscript-suites/playwright.config.ts

```diff
@@ -2,10 +2,10 @@ import { PlaywrightTestConfig, devices } from '@playwright/test';
 
 const config: PlaywrightTestConfig = {
   forbidOnly: !!process.env.CI,
-  globalTeardown: './e2e/autoscript-suites/teardown',
```

Comment:
moved into local package configuration. Allows for better ergonomics when we add a new package that uses a different playwright config.